tests/unit/executor/test_hubio.py

```diff
@@ -1403,3 +1404,13 @@ def _mock_post(url, json, headers=None):
     host, port = HubIO.deploy_public_sandbox(args)
     assert host == 'http://test_new_deployment.com'
     assert port == 4322
+
+
+@pytest.mark.parametrize('path', ['download_dummy_executor'])
+def test_list(mocker, monkeypatch, path):
+    exec_path = os.path.join(cur_dir, path)
+    args = set_hub_list_parser().parse_args([])
+
+    with monkeypatch.context() as m:
+        m.setattr(hubio, 'list_local', lambda: [Path(f'{exec_path}/latest.dist-info')])
+        HubIO(args).list()
```

Comment:
this test is a bit unclear. Can we somehow verify result of it?

Reply:
Well, it just prints the result to the user, so we can't get the result to verify.

Reply:
Agree with @delgermurun  

You must verify the result somehow, otherwise how do you know it's working? We can't check the output manually each time.

Reply:
> You must verify the result somehow, otherwise how do you know it's working? We can't check the output manually each time.

Ok, let me think about it.